docs/compatibility.md

```diff
@@ -0,0 +1,15 @@
+# Compatibility with Previous Versions of MMDetection3D
+
+This document provides detailed descriptions of the BC-breaking changes in MMDetection3D.
+
+## MMDetection3D 0.12.0
+
+### SUNRGBD dataset for ImVoteNet
+
+We adopt a new pre-processing procedure for the SUNRGBD dataset in order to support ImVoteNet, which is a multi-modality method requiring both image and point cloud data. In previous versions of MMDetection3D, SUNRGBD dataset was only used for point cloud based 3D detection methods. In MMDetection3D 0.12.0, we add ImVoteNet to our model zoo, thus updating SUNRGBD correspondingly by adding image-related pre-processing steps. Specificly, we made these changes:
+
+- Fix a bug in the image file path in meta data.
+- Convert calibration matrices from double to float to avoid type mismatch in further operations.
+- Add instructions in the documents on preparing image data.
+
+Please refer to the SUNRGBD [README.md](../data/sunrgbd/README.md) for more details.
```

Comment:
Should use absolute path instead of relative path because users may click it online? (I remember Wenwei @ZwwWayne told me this before, is that correct?)

Reply:
Yes, we'd better use the link in readthedocs.

Reply:
If you must use link to some files, please also use the absolute path, because when browsing the web-based documentation, the relative path can not be found.

Reply:
then shall I change every relative path to absolute path in the docs?